servers/quarkus-server/src/main/java/com/dremio/nessie/server/config/ApplicationConfig.java

```diff
@@ -117,13 +118,25 @@ public VersionStoreDynamoConfig getVersionStoreDynamoConfig() {
     @ConfigProperty(name = "initialize", defaultValue = "false")
     boolean isDynamoInitialize();
 
-    @ConfigProperty(name = "refTableName", defaultValue = "nessie_refs")
+    @ConfigProperty(defaultValue = DynamoStoreConfig.REF_TABLE)
```

Comment:
shall we add defaults to the property instead of adding them to `application.properties`?